app/controllers/api/v1/api_controller.rb

```diff
@@ -0,0 +1,12 @@
+module Api
+  module V1
+    class ApiController < ApplicationController
+      before_action :authenticate_api_v1_user!
+      # rubocop:disable Rails/LexicallyScopedActionFilter
+      skip_before_action :authenticate_api_v1_user!, only: [:create]
```

Comment:
This should only be on the appropriate controller, not for all the create methods